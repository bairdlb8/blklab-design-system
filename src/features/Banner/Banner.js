import React from 'react'
import PropTypes from 'prop-types'

import './Banner.scss'

const Banner = ({className=''}) => {
  return <div>Banner</div>
}

Banner.propTypes = {}

export default Banner
