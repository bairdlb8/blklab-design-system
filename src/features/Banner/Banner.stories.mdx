import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Banner from './Banner'

<Meta title="Features / Banner" component={ Banner } />

# Banner

## Add some text here

export const Template = (args) => {
  return <Banner {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Banner } />
