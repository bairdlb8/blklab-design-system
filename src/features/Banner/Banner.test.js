import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import Banner from './Banner'

expect.extend(toHaveNoViolations)

describe('<Banner />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Banner />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Banner />)

    expect(asFragment(<Banner />)).toMatchSnapshot()
  })

})
