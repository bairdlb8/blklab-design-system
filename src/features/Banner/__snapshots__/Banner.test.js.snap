// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Banner
  </div>
</DocumentFragment>
`;
